tests/drivers/can/api/testcase.yaml

```diff
@@ -14,12 +14,7 @@ tests:
       - CONFIG_CAN_ACCEPT_RTR=y
   drivers.can.api.twai:
     extra_args: DTC_OVERLAY_FILE=twai-enable.overlay
-    filter: dt_compat_enabled("espressif,esp32-twai")
     platform_allow:
-      - esp32_devkitc_wroom/esp32/procpu
-      - esp32_devkitc_wrover/esp32/procpu
-      - esp32c3_devkitm
-      - esp32s2_saola
       - esp32s3_devkitm/esp32s3/procpu
       - xiao_esp32s3/esp32s3/procpu
```

Comment:
We can remove the filter, but I'd rather keep the `platform_allow` list (since `twai-enable.overlay` is not guaranteed to work with all ESP32* boards) and add an `integration_platforms` key with just one of these boards.

Reply:
none of those targets support can (they do not have supported: can), so they were never used/buit and get filtered out.